carbon-design-system/carbon
# fix(Datepicker): add aria role to Flatpickr dialog

Closes #7246 

Flatpickr's calendar dialog is not rendered in a landmark causing an error with IBM Equal Access Accessibility Checker. This adds an aria role and an aria label to the container div on the calendar dialog. 

#### Changelog

- `addRoleAttributeToDialog` function

#### Testing / Reviewing

1. Open a **"single"** Datepicker in the Storybook.
2. Run IBM Equal Access Accessibility Checker
3. See no violations
4. Click the input to trigger the calendar dropdown
5. See no violations
---
1. Open a **"range"** Datepicker in the Storybook
2. Run IBM Equal Access Accessibility Checker
3. See no violations
4. Click the input to trigger the calendar dropdown
5. See no violations
